### DISCUSSION TITLE
JCB and yootheme

### DISCUSSION THREAD

**Discussion by dormasta** [joomengine/Joomla-Component-Builder#912] [Q&A]
Hi,

Does anyone know about a tutorial about how to integrate Yootheme into a JCB generated component or the other way around? 
1) In other theme frameworks there is a place holder like “component area” in helix. How does it work in yootheme?
2) Is there a smart way to get the generate template style into the “site view”?

thanks for your help, cheers

**Comment by Llewellynvdm** [comment]
Since Yootheme use Uikit, adding Uikit (selecting it as a library) to your site view and using those CSS classes in your html will work perfectly with Yootheme templates.

That is all JCB does out of the box at this time towards Yootheme... anything more you will have to hand code.

**Comment by MacJoom** [comment]
It's all about providing dynamic content. I have adapted an yootheme extension working as a bridge to phocacart (https://github.com/MacJoom/yootheme_phocacart) providing dynamic content (products and product categories as dynamic content). It should be possible to generate such an extension in jcb but this is only after the the transistion to joomla 4 has been made.
